src/app/user-update/user-update.component.html

```diff
@@ -30,7 +30,7 @@ <h3 mat-dialog-title>{{language?.messages.heading}}</h3>
                [formControl]="updateForm.controls['lastName']">
       </div>
       <p id="error"
-         *ngIf="updateForm.controls['firstName'].hasError('required') && (updateForm.controls['firstName'].touched && !isFocused)">
+         *ngIf="updateForm.controls['firstName'].hasError('required') && (updateForm.controls['firstName'].touched)">
```

Comment:
What is the reason for making this change?

Reply:
yes and even if this change was made, it should not be sent with this PR.

Reply:
@lunayach please address this

Reply:
@shreyans29 @Raxerz to enable caching, we are required to make the production build. And in the production build, we required to remove even smaller of lint errors. So, that's why it was removed.

Reply:
@lunayach how is this a lint error?

Reply:
@lunayach need to address this comment, because lint errors do not occur in the HTML files.. was there a lint error on isFocused ?

Reply:
@Raxerz @shreyans29 in HTML we had **isFocused( )** method, that was not defined in *.ts* file. That was causing the error.

Reply:
I remember while modifing the registration page you had removed the is focused method. So check if it really not needed, if not then fine otherwise add it.

Reply:
Ya! @shreyans29, **isFocused( )** was never used and that's why it is not defined as well.